Sources/Controllers/Settings/ImportExport/OAPluginInstalledViewController.mm

```diff
@@ -287,9 +287,14 @@ - (IBAction)onDisablePressed:(UIButton *)sender
     {
         OAProduct *product = [[OAIAPHelper sharedInstance] product:_pluginId];
         if (product)
+        {
             [_iapHelper disableProduct:_pluginId];
+            [_app.data setMapillary: NO];
```

Comment:
Remove space. Will this disable the mapillary layer regardless of which plugin is disabled?